src/ApiTestCase.php

```diff
@@ -429,6 +429,6 @@ private function assertSourceExists($source)
      */
     private function getRootDir()
     {
-        return $this->get('kernel')->getRootDir();
+        return $this->get('kernel')->getProjectDir();
```

Comment:
Not sure, how we should handle it, as this is not exactly the same path https://github.com/symfony/symfony/issues/29110#issuecomment-436327589

Reply:
I'd suggest releasing it with a new version and adding it to the `CHANGE.MD` to outline the edge cases, where something needs to be changed.